test/loadtest/readme.md

```diff
@@ -0,0 +1,75 @@
+# Distributed load testing using Bedrock kubernetes Cluster and Locust
+
+Load testing is crucial to any application's success. We have chosen to use Locust as a tool for facilitating the load testing of the solution you have deployed in Bedrock. In order to begin load testing your application, make sure you have a cluster to deploy the Locust load tester on. If you do not have a load-testing cluster, follow the directions in [azure-simple](../../cluster/environments/azure-simple/) and use that cluster when you execute the following steps.
+
+## Setup
+
+1. Clone or download `Bedrock` git repo and navigate to `test/loadtest` folder.
+2. Update the `locust-tasks\tasks.py` file specific to however you would like your load test to be run. More details about how to write the tasks file can be found in the [Locust documentation](https://docs.locust.io/en/stable/writing-a-locustfile.html). All files in `locust-tasks` will be mounted to POD.
+3. Build docker image and store it in your container registry such as docker or Azure Container Registry.
+    * Make sure you are logged into Azure subscription
+
+        ```az login```
+
+	* Login to your desired Azure Container Registry using 
+    
+        ```az acr login --name <ACR_NAME>```
+
+	* Build your image by running
+
+        ```docker build . -t locust```        
+
+	* Tag your image in order to push it up to the registry: 
+    
+        ```docker tag <IMAGE_HASH> <ACR_NAME>.azurecr.io/<DESIRED_CONTAINER_NAME>:<DESIRED_VERSION_NUMBER>```
+
+	* Push to your registgry by running 
+    
+        ```docker push <ACR_NAME>.azurecr.io/loadtesting/<DESIRED_CONTAINER_NAME>:<DESIRED_VERSION_NUMBER>```
+
+4. Deploy Locust onto your Cluster
+
+    * Ensure that `kubectl` is set to connect to your load testing cluster.
+    * Make script file executable
+
+        ```chmod +x deploy_locust.sh```
+
+    * Run `deploy_locust.sh` by specifying values for the following parameters. Here is an example:
+
+        ```./deploy_locust.sh -r "myDockerRegistry/locust" -t "0.1.0" -h "http://servicetotest.com"  -w "7" ```
```

Comment:
Add a note here that this url is the root of the application under test and that tasks.py will define which route is hit.